src/components/AppLayout/index.tsx

```diff
@@ -223,18 +179,17 @@ const AppLayout = ({ className, Component, pageProps }: Props) => {
 		items = [...gov1Items.overviewItems, getSiderMenuItem('Alliance', 'alliance_group', null, [
 			...gov1Items.allianceItems
 		]), getSiderMenuItem('Fellowship', 'fellowship_group', null, fellowshipItems)];
-		collapsedItems = [...gov1Items.overviewItems, ...gov1Items.allianceItems, ...fellowshipItems];
 	} else if (network === AllNetworks.WESTENDCOLLECTIVES) {
 		items = [...gov1Items.overviewItems, getSiderMenuItem('Alliance', 'alliance_group', null, [
 			...gov1Items.allianceItems
 		])];
-		collapsedItems = [...gov1Items.overviewItems, ...gov1Items.allianceItems];
```

Comment:
why we remved this

Reply:
because we no longer need Collapsed items of Gov1 cuz now Gov2 will show all the links of Gov1